main.cpp

```diff
@@ -222,11 +228,14 @@ struct Piano
     double pianoSize = 20.80;
     std::string brandName = "Yamaha";
     int totalNumPedals = 3;
+    int numKey = 40;
+    double playTime = 42.25;
+    double notePlayed;
 
     Piano();
     ~Piano();
 
-    double playNote(int numKey, double playTime = 45.0);
+    double playNote(int mNumKey, double mPlayTime = 45.0);
     int muteNote(bool playOrMute = false, bool stringPadOn = true);
     void volumeDown(bool quietMode = true, int numPedal = 2);
     void arpeggio(int keyNum, int range, int interval);
```

Comment:
`arpeggio` is not an action/verb.  it is a noun. 
`arpeggiate` is a verb.
use verbs in the names of functions
- [ ] revise